docs/docs/software-architecture.md

```diff
@@ -173,14 +181,15 @@ The software architecture supports the following quality requirements:
 3. Each feature of the Swapchain platform is documented in a subdomain of Github (Github, 2020) by following this link: https://chronark.github.io/swapchain [1]
 4. The Swapchain platform will be available 24 hours a day, 7 days a week.
```

Comment:
We have no control over this. Netlify can be down and even a blockchain node that we are using can be down.